### PR TITLE
[Role Datums] BASIC Gamemode Datums

### PR DESCRIPTION
**Pull request by Intigracy**
Sorry this took so long and isn't finished, I really lost motivation.

Also moved everything done so far into one folder under /datums so people who work further on role datums as a whole don't have to jump around the code as much.

won't compile obviously

### CODE REVIEW

**Comment by jknpj** [comment]
our guy

File: vgstation13.dme [hunk 1]
```diff
@@ -2178,7 +2156,6 @@
 #include "code\WorkInProgress\buildmode.dm"
 #include "code\WorkInProgress\explosion_particles.dm"
 #include "code\WorkInProgress\periodic_news.dm"
-#include "code\WorkInProgress\Cael_Aislinn\sculpture.dm"
```
Comment (Intigracy) [inline vgstation13.dme]: not sure why this and mine_vending were changed, any ideas?

Comment (Exxion) [reply vgstation13.dme]: I dunno man it seems like the PR author would be the one to ask about that

Comment (Intigracy) [reply vgstation13.dme]: I'll ask him and see what he says.

Comment (DamianX) [reply vgstation13.dme]: mine_vending isn't a thing anymore on Bleeding-Edge

Comment (Intigracy) [reply vgstation13.dme]: That'll get resolved when this gets merged onto bleeding-edge anyway

**Comment by Kurfursten** [comment]
>taking forever to finish something due to motivation

Most relatable PR of the year

**Comment by DrSnips** [comment]
Looks like a compile failure, throw it in the trash. We surely don't need any of this.

**Comment by Intigracy** [comment]
merging so robo can fix his pr

**Pull request merged**